cmd/generator/bundles.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/mattermost/mattermost-marketplace/internal/model"
+)
+
+// addPlatformSpecificBundles includes the platform-specific bundle URLs and signatures in the Marketplace entries.
+func addPlatformSpecificBundles(plugin *model.Plugin) (*model.Plugin, error) {
+	if plugin.HomepageURL == "" {
+		return plugin, nil
+	}
+
+	slashIndex := strings.LastIndex(plugin.HomepageURL, "/")
+	repo := plugin.HomepageURL[slashIndex+1:]
```

Comment:
E.g. MSCalender uses not the GitHub page as a `HomepageURL` but instead uses a GitBook URL. Can `addPlatformSpecificBundles` just accept a repo and org?

Reply:
@hanzei I was using this to figure out the repo name, which we don't have a source of truth for in `plugins.json` at the moment. I am adding a `RepoName` field to the plugin entry so we have it available. We can backfill these in of course.

The plugins are being stored in s3 as their repo name, as opposed to their plugin id. For example:

`com.mattermost.mscalendar` is being served at `https://plugins-store.test.mattermost.com/release/mattermost-plugin-mscalendar-v1.0.0.tar.gz`.

So the raw repo name is what is needed to calculate this URL. I suggest storing it in the top level of the marketplace entries so the manifests don't need to be updated for each existing plugin, though we could start putting it there as well.